sharktuner/sharktuner/candidate_ordering.py

```diff
@@ -0,0 +1,105 @@
+from enum import Enum
+from typing import Type, Optional, Callable, Iterable, Any
+from . import common
+import random
+import math
+import logging
```

Comment:
```suggestion
from typing import Type, Optional, Callable, Iterable, Any
import random
import math
import logging

from . import common
```

import local modules after standard library imports.

Reply:
[8c587f0](https://github.com/nod-ai/shark-ai/pull/2555/commits/8c587f06bf40b11ff46fb4a1fbd5c99bc3dfe5d5)